src/breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types';
+import {
+  BreadcrumbItemStyles,
+  BreadcrumbButtonStyles,
+  BreadcrumbLinkStyles,
+  BreadcrumbWrapper
+} from './Breadcrumb.styles';
+
+const Breadcrumb = (props) => {
+  const { className, component, href, children, dataTestId, onClick, ...rest } =
+    props;
+
+  let as = 'a';
+  let customComponentProps = {};
+  if (component) {
+    as = component;
+    customComponentProps = { ...rest };
+  }
+
+  if (href) {
```

Comment:
I dont think this will work - because if we use `NavLink` from react router there will not be an href prop (because it will use `to` instead). 

Take a look at the LinkButton - it always renders the same component regardless of a custom component or not. Also - there may be a scenario where both onClick and the href prop can be set

Reply:
Alright I changed it to render the same component that makes a lot of sense.
If theres an onClick present, does it matter if the html element ends up still being an <a>? It seems to work fine but wondering thats a no-go for accessibility.